client/tailwind.config.js

```diff
@@ -2,7 +2,11 @@
 module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        "speakers-corner-blue": "#0077c0",
```

Comment:
nit: would be good to have a shorter name for this. Maybe brand-blue or sc-blue or even just primary if this is the primary color being used in the site

Reply:
shortened it to "sc-blue"